tools/clang/include/clang/SPIRV/AstTypeProbe.h

```diff
@@ -324,6 +324,12 @@ bool isOrContainsNonFpColMajorMatrix(const ASTContext &,
                                      const SpirvCodeGenOptions &, QualType type,
                                      const Decl *decl);
 
+/// \bried Returns true if the given type is `vk::ext_result_id<T>`.
```

Comment:
nit: Looks like a copy-paste error from below, these should all be "\brief"

Reply:
Nice catch!